backend/app/api/routes/download.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import FastAPI, HTTPException, APIRouter
+from fastapi.responses import FileResponse
+
+from sqlmodel import select, Session, col
+from app.api.deps import SessionDep
+from app.models import Document
+from app.repositories import document_repo
+import os
+
+router = APIRouter()
+
+@router.get("/documents/{doc_id}/download")
+def download_file(
+    doc_id: int,
+    session: SessionDep
+):
+    doc = session.get(Document, doc_id)
+    if not doc:
+        raise HTTPException(status_code = 404, detail = "File not found")
+    
+    DATA_PATH = "../data"
```

Comment:
Read the data path value from: https://github.com/pingcap/autoflow/blob/db1471d48ddb3fe9a870c01fa7e22a853054db60/backend/app/file_storage/local.py#L8

Reply:
doned